SUPPORT.md

```diff
@@ -0,0 +1,11 @@
+# Support
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
+feature request as a new Issue.

```

Comment:
This is incorrect. vcpkg-tool only has PR activated.

Reply:
Yeah I stripped the whole file now, thanks for pointing it out!